src/game/special_stage/physics.c

```diff
@@ -98,10 +98,14 @@ static void Task_PhysicsMain(void)
 
         sub_80047A0(arrowAngle, 256, 256, 30);
 
-        if (stage->paused == FALSE) {
+        if (stage->paused == FALSE)
             UpdateSpriteAnimation(&player->roboArrow);
-        }
-        DisplaySprite(&player->roboArrow);
+
+#if PORTABLE
```

Comment:
To be clear, this isn't a bug, it's because in widscreen the arrow is in the "center" of the screen